dpe/src/dpe_instance.rs

```diff
@@ -42,7 +42,7 @@ pub struct DpeEnv<'a, T: DpeTypes + 'a> {
 }
 
 #[repr(C, align(4))]
-#[derive(AsBytes, FromBytes, Zeroize)]
+#[derive(IntoBytes, TryFromBytes, KnownLayout, Immutable, Zeroize)]
```

Comment:
@jhand2 FYI I believe this was the only public change. 

In order to trim down the enums, `DpeInstance` needed to switch from `FromBytes` to `TryFromBytes`.